handlers.py

```diff
@@ -47,7 +47,22 @@
 <meta charset="utf-8">
 <title>$title</title>
 <style type="text/css">
-.u-uid { display: none; }
+body {
+  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+}
+.u-uid {
+  display: none;
+}
+.u-photo {
+  max-width: 50px;
+  -webkit-border-radius: 4px;
+  -moz-border-radius: 4px;
+  border-radius: 4px;
```

Comment:
looks like this has been supported in webkit and firefox for a long time: http://caniuse.com/#feat=border-radius . maybe drop the vendor prefixed lines?